tests/sentry/incidents/endpoints/test_organization_alert_rule_index.py

```diff
@@ -856,7 +856,7 @@ def test_team_filter_no_access(self):
             response = self.client.get(
                 path=self.combined_rules_url, data=request_data, content_type="application/json"
             )
-        assert response.status_code == 400
```

Comment:
The PR description mentions that this previously responded with a 500 response. Is that reproduceable?

Reply:
Sorry I wasn't specific in the description, 500 on the eventsv2 endpoint. This 400 is raised here: https://github.com/getsentry/sentry/blob/461c09ff19deb869847f99e39be625b24cb04d1f/src/sentry/incidents/endpoints/organization_incident_index.py#L89
The 500 can be reproduced on the team stats page when you try to request stats for a team you aren't a part of.

Reply:
(I was able to reproduce by loading the team stats page, on a new tab leaving a team and then selecting that team I just left in the team stats page_